src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -78,11 +78,14 @@
 short-summary: Creates an Azure Container Registry.
 examples:
   - name: Create a managed container registry with the Standard SKU.
-    text: >
+    text: >-
         az acr create -n MyRegistry -g MyResourceGroup --sku Standard
   - name: Create an Azure Container Registry with a new storage account with the Classic SKU (Classic registries are being deprecated by March 2019).
-    text: >
+    text: >-
         az acr create -n MyRegistry -g MyResourceGroup --sku Classic
+  - name: Creates an Azure Container Registry. (autogenerated)
```

Comment:
This is the same as the existing example except with a different SKU value and using long instead of short options.